### ISSUE TITLE
Duda Test cases 27

### ISSUE THREAD

**Issue by CataAlamos** [RodrigoToroIcarte/IIC2113-2023-1#97]
¿Para jugar Back Body Drop mi ultima carta en mi RingArea debe ser Irish Whip? Porque por enunciado se dice: "The card titled Irish Whip must be played before playing this card. When successfully played, you may either draw 2 cards, or force opponent to discard 2 cards.", y en mi test case 27 mi jugador juega Irish Whip, luego termina su turno y despues no vuelve la opción de jugar Back Body Drop.

**Comment by RodrigoToroIcarte** [comment]
No. Eso es lo bonito/complicado del `Irish Whip`. La carta `Back Body Drop` solo puede ser jugada justo después de un `Irish Whip` (en el mismo turno o como la primera carta jugada luego de revertir un `Irish Whip` con otro `Irish Whip`). Por lo mismo, no es suficiente con ver que la última carta en mi `ring area` sea un `Irish Whip`.

La carta `Irish Whip` fue descrita como una de las dificultades del proyecto en una clase:
![Screenshot from 2023-06-05 16-11-50](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/7000518/16af690d-7d54-49df-8ae4-5669e4b1cfa4)
